Makefile

```diff
@@ -199,10 +194,10 @@ docker-linux-arm:
 	$(call build-tags) && \
 	CGO_ENABLED=1 CC=arm-linux-gnueabi-gcc CXX=arm-linux-gnueabi-g++ CGO_ENABLED=1 GOOS=linux GOARCH=arm GOARM=7 go build -a -o lantern_linux_arm -tags="$$BUILD_TAGS" -ldflags="$(LDFLAGS) $$EXTRA_LDFLAGS -linkmode internal -extldflags \"-static\"" github.com/getlantern/flashlight/main
 
-docker-windows-386:
+windows-386: require-assets
```

Comment:
I don't think we benefit a lot from this. It already happened some months ago that I was running a beta version of Go locally (in OSX, which we cannot build inside docker) that created binaries that crashed upon starting, nobody had the same problem as me because they haven't updated Go versions to the latest beta, I don't run Go betas anymore. Using the docker image prevents the same problems on Windows and Linux, we can be sure we have exactly the same build. We cannot do that for OSX and that's a shame, but we can be on the safe side with Linux and Windows (Windows being the most used) and I really think that saves debugging time.